docs/source/examples.rst

```diff
@@ -3,4 +3,12 @@ Examples
 
 
 Example 1
-=========
\ No newline at end of file
+=========
+This example comes from Section 6 of the `GP Fitting <http://web.mit.edu/~whoburg/www/papers/gp_fitting.pdf>`_, by W. Hoburg et. at.
```

Comment:
1. "of the `GP Fitting" should be "of `GP Fitting", or "of the paper `Fitting geometric programming models to data"
2. et. at --> et. al.